client/styles/custom.scss

```diff
@@ -55,3 +55,7 @@ $tablet: (
   clear: both;
   height: 0;
 }
+
+.hide {
+  display: none !important;
```

Comment:
It's totally fine for a prototype. So often it's the choice we have to make with CSS